libs/react/icons/src/lib/components/arrowLeftIcon.tsx

```diff
@@ -0,0 +1,4 @@
+import svg from '../../../../../../libs/expo/shared/icons/src/assets/third_party/mingcute_design/svg/mingcute:arrow-left-line.svg';
```

Comment:
Hey @Davit-BetterAngels 

**Environment**
I tested this on mobile (Safari and Chrome) and desktop (Chrome), used 301 S. Flower St. Los Angeles, CA -
I found 4 green pins on the map on desktop.

**Mobile:**

1. Safari - When I used the address above, I didn't see any result. I also used the URL (https://shelter.dev.betterangels.la/view-shelter/shelter/11) directly on my Safari but nothing shows up on the page. (See the attachment 1)
2. Chrome - it displays as the same as the desktop Chrome browser
3. The phone number should be formatted e.g.,(323)-234-4455
4. The font size seems to be very small (14px ones) - Can we change it to 1rem?
5. As for Share icon, you can find the icon (20px) [here](https://www.figma.com/design/waIeX8dknHKedTrVxCQAAG/Outreach-App?node-id=20237-148191&t=X6J0JRR0U8VEwzFc-4). The whole 20px icon set is [here](https://www.figma.com/design/waIeX8dknHKedTrVxCQAAG/Outreach-App?node-id=20265-150110&t=X6J0JRR0U8VEwzFc-4).
6. I see a big white gap at the bottom before the footer (See attachment 6)

<img src="https://github.com/user-attachments/assets/6ad4d868-c3c3-4cc8-8607-579be6941525"  width="320" />

<img width="320" alt="image" src="https://github.com/user-attachments/assets/9a9f7f19-cdad-4a8b-aae7-ba7baf476364" />


Thanks!

Reply:
As for my comment No.7, I found the ticket DEV-1081. But it's not working on my end.